libs/eo/shared/assets/src/assets/configuration/eo-api-environment.local.json

```
@@ -1,3 +1,3 @@
 {
-  "apiBase": "https://eo-u-privacypolicy.westeurope.cloudapp.azure.com/api"
+  "apiBase": "https://eo-u-landingpage.westeurope.cloudapp.azure.com//api"
```

issue: Remove duplicate slash.
```suggestion
  "apiBase": "https://eo-u-landingpage.westeurope.cloudapp.azure.com/api"
```